.github/workflows/post-merge-kms-dev.yml

```diff
@@ -36,9 +36,7 @@ jobs:
           aws-region: ${{ env.AWS_REGION }}
 
       - name: SAM Validate
-        run: sam validate --region ${{ env.AWS_REGION }} -t 
-          working-directory: ./infra-l2-kms/**
-          template-file: template.yaml
+        run: sam validate --region ${{ env.AWS_REGION }} -t infra-l2-kms/template.yaml
```

Comment:
update working dir on line 24, and remove the path here.
`working-directory: infra-l2-kms`